src/archemist/core/optimisation/recipe_converter.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import yaml
+from archemist.core.persistence.yaml_handler import YamlHandler
+from nested_lookup import nested_update
+from pathlib import Path
+import logging
+import re
+
+class RecipeUpdate:
+    def __init__(self, template_path, recipe_path) -> None:
+        self.recipe_dict  = YamlHandler.load_recipe_file(template_path)
+        self.recipe_dir = recipe_path
+        self.placeholders = []
+        self.placeholder_keys_all = []
+        self.placeholder_keys_any = []
+        self.find_placeholders(self.recipe_dict)
+
+    def find_placeholders(self, recipe_dict):
+        if isinstance(recipe_dict, dict):
+            for key, value in recipe_dict.items():
+                if isinstance(value, str) and re.search(r"^{{*.*}}$", value):
+                    _strings = re.split("\.", value)
+                    _data_type = _strings[0].lstrip("{")
+                    _out_type = _strings[1].rstrip("}")
```

Comment:
To be consistent with the style of the rest of the library please refrain from using "_variable" with temporary variable names, as if we follow that style we would need to apply to all of the code base. Please apply this comment to the reminder of the code

Reply:
yes, modified